v03_pipeline/lib/tasks/base/base_delete_table.py

```diff
@@ -0,0 +1,19 @@
+import hailtop.fs as hfs
+
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.tasks.base.base_hail_table import BaseHailTableTask
+from v03_pipeline.lib.tasks.files import GCSorLocalFolderTarget, GCSorLocalTarget
+
+logger = get_logger(__name__)
+
+
+class BaseDeleteTableTask(BaseHailTableTask):
+    def complete(self) -> bool:
+        logger.info(f'DeleteTableTask: checking if {self.output().path} exists')
+        return (
+            not GCSorLocalTarget(self.output().path).exists()
+            and not GCSorLocalFolderTarget(self.output().path).exists()
```

Comment:
Would there ever be a case where `GCSorLocalTarget(self.output().path).exists()` is false but `GCSorLocalFolderTarget(self.output().path).exists()` is true?

Reply:
yes!  The "Folder" target checks that there's a `_SUCCESS` file inside the path.  So in many cases they do not give the same answer.